firebase-dataconnect/src/main/kotlin/com/google/firebase/dataconnect/serializers/TimestampSerializer.kt

```diff
@@ -71,9 +71,7 @@ public object TimestampSerializer : KSerializer<Timestamp> {
     val regex =
       Regex("^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}(\\.\\d{0,9})?(Z|[+-]\\d{2}:\\d{2})$")
 
-    require(strUppercase.matches(regex)) {
-      "Value does not conform to the RFC3339 specification (str=$str)"
```

Comment:
RFC3339 supports more than 9 digits in time-secfrac, so what is allowed by Firebase is not 100% RFC3339